swagger/swagger.yaml

```diff
@@ -44,4 +44,16 @@ paths:
       x-tags:
       - tag: "default"
 
+  /rackmount:
+    get:
```

Comment:
host, user and password should be passed to this route.  This will allow the route to log into the UCS service and not use the global handle.